### ISSUE TITLE
Lister les traductions par thème

### ISSUE THREAD

**Issue by goulvench** [pompage/site#4]
Parce qu'on commence à en avoir un paquet par thème.

Si on supprime complètement la page "Toutes les traductions", mettre un moteur de recherche.

Il s'agit de créer une page par thème qui liste toutes les traductions correspondantes.

**Comment by arkhi** [comment]
Tu veux parler de http://www.pompage.net/articles_par_theme ?

**Comment by goulvench** [comment]
Oui, mais en faisant une page par thème pour que les pages soient moins
lourdes et plus faciles à consulter sur mobile.

## 

Goulven Champenois
